README.md

```diff
@@ -31,11 +31,11 @@
 
 <!-- PROJECT DESCRIPTION -->
 
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you remove this unnecessary description under the Headers
<img width="697" alt="image" src="https://user-images.githubusercontent.com/52934128/217167132-94dcd3bf-6254-4699-a6d1-1321ba09f1cb.png">